trigger/build.go

```diff
@@ -269,6 +270,11 @@ func (t *buildTriggers) makeData(sub *event.Subscription, pastTenseOverride stri
 	projectName := t.build.Project
 	if api.ProjectIdentifier != nil {
 		projectName = utility.FromStringPtr(api.ProjectIdentifier)
+	} else {
+		identifier, err := model.GetIdentifierForProject(*api.ProjectId)
```

Comment:
This fallback logic seems to be identical to what `api.BuildFromService` already does internally, which [sets the `ProjectId` based on the build's project](https://github.com/evergreen-ci/evergreen/blob/24b4506727504198dd5cbbd47fb020c3e7e0a89a/rest/model/build.go#L90), then [uses `model.GetIdentifierForProject` to get the identifier](https://github.com/evergreen-ci/evergreen/blob/24b4506727504198dd5cbbd47fb020c3e7e0a89a/rest/model/build.go#L117-L122).

Reply:
removed